hasher-matcher-actioner/src/OpenMediaMatch/storage/postgres/impl.py

```diff
@@ -617,6 +618,8 @@ def bank_yield_content(
         # Query for all ContentSignals and stream results with the proper batch size
         query = (
             select(database.ContentSignal)
+            .join(database.BankContent)
+            .join(database.Bank)
```

Comment:
(answer to previous question) - two additional joins - does this appreciably impact the iteration speed?

Reply:
We have index on the tables, but there's definitively an impact. I'll have to calculate at 10M records tho 🤔